lib/pages/home/bottom_sheets/confirm_action.dart

```diff
@@ -76,8 +76,7 @@ class OBConfirmActionBottomSheetState
     List<Widget> columnItems = [
       OBText(
         confirmationText,
-        size: OBTextSize.extraLarge,
-        style: TextStyle(fontWeight: FontWeight.bold),
+        style: TextStyle(fontWeight: FontWeight.bold, fontSize: 24),
```

Comment:
Why did you change this from `OBTextSize.extraLarge` to the constant `24`?
(Not necessarily an issue, I'm just curious as to why.)

Reply:
extra large is too large and large is too small 😢